issue.go

```diff
@@ -1363,6 +1370,8 @@ func InitIssueWithMetaAndFields(metaProject *MetaProject, metaIssuetype *MetaIss
 			issueFields.Unknowns[jiraKey] = Option{
 				Value: value,
 			}
+		case "id":
+			issueFields.Unknowns[jiraKey] = []map[string]string{{"id": value}}
```

Comment:
This looks like it should be a single value and not an array, i.e `map[string]string`, not `[]map[string]string`, since there is already a `case "array"` section above.

Also, can't we write this using the `Id` type? 
```go
issueFields.Unknowns[jiraKey] = []Id{
        {Id: value}
}
```

Reply:
you are correct! I've been silly here and was working with a a clone of the orginal go-jira for my testing with kaiju and had the below here but must of copied across the wrong thing to the fork of the repo 
```
		case "id":
			issueFields.Unknowns[jiraKey] = Id{
				Id: value,
			}
```